core/src/main/scripts/importer/validateData.py

```diff
@@ -1346,6 +1348,30 @@ def checkSwissProt(self, value):
                 return True
         # if no reasons to return with a message were found, return valid
         return True
+    
+    def checkStartPosition(self, value):
+        """Check that the Start_Position value is an integer."""
+        if value.isdigit() == False or (value.isdigit() and '.' in value):
+            self.logger.error(
+                'The start position of this gene is not '
```

Comment:
@fedde-s yeap, it's the mutation position! Completely my fault, I'm going to change that.